packages/ignition/src/app/components/documentation/sidebar-left.tsx

```diff
@@ -87,7 +87,7 @@ export const SidebarLeft = ({ lang }: PropsWithLang) => {
                 : docs.filter(
                     (doc: any) =>
                       doc[1][lang].group === mainDocEnSlug &&
-                      doc[1][lang].group != doc[1][lang].slug
+                      doc[1][lang].group != doc[1][lang].enslug
```

Comment:
I'm going to believe on you on this one. No clue

Reply:
This was causing the bug of the lone menu being rendered at the end of Usage. It was looking for a parent "usando" which was never found in pt-br.